README.md

```diff
@@ -105,8 +105,8 @@ we deviate from normal SemVer in the following ways:**
 
 ## License
 
-Font Awesome Free is free, open source, and GPL friendly. You can use it for
-commercial projects, open source projects, or really almost whatever you want.
+Font Awesome Free is a free, open-source, and GPL friendly. You can use it for
```

Comment:
Using the article `a` here turns `free, open-source, and GPL friendly` into adjectives without a subject.